app/javascript/packs/Notification/notification.jsx

```diff
@@ -47,13 +54,43 @@ export default class Notification extends React.Component {
             );
         }
     }
+
     importanceCheck() {
         if (this.props.importance == 'trivial') 
-            return '!'
+            return '!';
         else if (this.props.importance == 'important')
             return '!!!'
     }
 
+    handlePhotoUrls() {
+        this.setState({ isLoading: true });
+        fetch('/post_events/'+ this.props.NotificationID +'.json')
+            .then(response => response.json())
+            .then(posts_events => {
+                this.setState({ photo_urls: posts_events.images_url, isLoading: false});
+                console.log(this.state.photo_urls[0].url)
+            });
+    }
+
+    loadImages() {
+        return this.state.photo_urls.map((photo, index) =>
+            <Card style={{ width: '15rem' }}>
+                <Card.Body>
+                    <Image src={ "http://localhost:3000"+ photo.url } value={photo.url} onClick={() => this.showZoomInPhoto(photo.url)} fluid/>
+                    <Button href={"http://localhost:3000/post_events/download/" + this.props.NotificationID +"/"+index} target="_blank"> Download </Button>
```

Comment:
consider using https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/template_strings for string concatenation